clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/unsecured/OAuthBearerUnsecuredLoginCallbackHandler.java

```diff
@@ -182,6 +182,14 @@ public void close() {
     private void handleTokenCallback(OAuthBearerTokenCallback callback) {
         if (callback.token() != null)
             throw new IllegalArgumentException("Callback had a token already");
+        if (moduleOptions.isEmpty()) {
+            log.info("Token not provided, this login cannot be used to establish client connections");
```

Comment:
Does this get logged on the broker side? Will that be too noisy?

Reply:
@junrao Thanks for the review, I have changed all the new log entries to debug. We also have https://issues.apache.org/jira/browse/KAFKA-7478 to reduce the level of some of the existing log entries.